### PR TITLE
replaced _.extend calls by _.clone calls

### PR DESCRIPTION
**Pull request by quentinlampin**
solves issue #1350

See https://github.com/angular-ui/angular-google-maps/issues/1350
for a discussion on lodash dependency and ways to get rid of it.

### CODE REVIEW

File: src/coffee/directives/api/models/child/marker-child-model.coffee [hunk 1]
```diff
@@ -23,7 +23,7 @@ angular.module('uiGmapgoogle-maps.directives.api.models.child')
         @trackModel = true, @needRedraw = false) ->
         #where @model is a reference to model in the controller scope
         #clonedModel is a copy for comparison
-        @clonedModel = _.extend({},@model)
+        @clonedModel = _.clone(@model,true)
```
Comment (nmccready) [inline src/coffee/directives/api/models/child/marker-child-model.coffee]: should it be cloneDeep?

Comment (nmccready) [inline src/coffee/directives/api/models/child/marker-child-model.coffee]: Or if you want to get away from lodash, angular.copy?

**Comment by nmccready** [comment]
Can you amend your commits to angularjs commit standards. I want to start using this since I have grunt-conventional-changlog now.

**Comment by nmccready** [comment]
https://github.com/angular/angular.js/blob/master/CONTRIBUTING.md#-git-commit-guidelines

File: src/coffee/directives/api/models/child/marker-child-model.coffee (hunk 1 above)
Comment (quentinlampin) [inline src/coffee/directives/api/models/child/marker-child-model.coffee]: Regarding the clone/cloneDeep choice, cloneDeep might be more readable but I settled for clone as it seems it was the one used before that lodash dep' thing, at least this is what the following comment suggests (src/coffee/directives/api/models/child/marker-child-model.coffee l.83): 
```
#changed from _.clone(model, true) eliminates lodash dep (so you can use underscore)
``` 

Btw, I have no opinion on the lodash debate, w/e works (cloneDeep untested, though).
You might want to ask the contributor that made the switch to _.extend.

**Comment by quentinlampin** [comment]
Ok, will do/try. I'm unsure whether or not  I can edit the commit message...
Help on that appreciated.

**Comment by quentinlampin** [comment]
Alright, did what I could (couldn't invest much time on git/hub yet). The last commit message should be fine. lines <99 characters, header/body/footer structure, etc.
Tell me if there is anything left to improve.

File: src/coffee/directives/api/models/child/marker-child-model.coffee (hunk 1 above)
Comment (nmccready) [inline src/coffee/directives/api/models/child/marker-child-model.coffee]: I was doing `_.extend {},` prior as it was to be a new total fresh object , thus clone deep. But if this fixes issues then I guess it is good. It is intended that the clone is a complete break from the original reference.

**Comment by nmccready** [comment]
LGTM

**Pull request merged**